tools/terraform-bundle/README.md

```diff
@@ -59,19 +59,26 @@ terraform {
 # Define which provider plugins are to be included
 providers {
   # Include the newest "aws" provider version in the 1.0 series.
-  aws = ["~> 1.0"]
+  aws = {
+    versions = ["~> 1.0"]
+  }
 
   # Include both the newest 1.0 and 2.0 versions of the "google" provider.
   # Each item in these lists allows a distinct version to be added. If the
   # two expressions match different versions then _both_ are included in
   # the bundle archive.
-  google = ["~> 1.0", "~> 2.0"]
+  google = {
+    versions = ["~> 1.0", "~> 2.0"]
+  }
 
   # Include a custom plugin to the bundle. Will search for the plugin in the
-  # plugins directory, and package it with the bundle archive. Plugin must have
-  # a name of the form: terraform-provider-*, and must be build with the operating
-  # system and architecture that terraform enterprise is running, e.g. linux and amd64
-  customplugin = ["0.1"]
+  # plugins directory (see hierarchy and package it with the bundle archive. Plugin must have
```

Comment:
I think there's a missing closing paren here: `(see hierarchy)`. But maybe you're referring to the "Plugins Directory Layout" below? I'm not sure what "see hierarchy" means.